src/Commands/core/DrupalDependenciesCommands.php

```diff
@@ -0,0 +1,307 @@
+                return array_map(function (string $dependencyString) {
+                    return Dependency::createFromString($dependencyString)->getName();
+                }, $extension->info['dependencies']);
+            }, $this->moduleExtensionList->reset()->getList());
+
+            if (!$notOnlyInstalled) {
+                $this->dependencies['module-module'] = array_intersect_key(
+                    $this->dependencies['module-module'],
+                    $this->installedModules,
+                );
+            }
+            if (!isset($this->dependencies['module-module'][$module])) {
+                throw new \InvalidArgumentException(dt('Invalid @module module', [
+                    '@module' => $module,
+                ]));
+            }
+        } elseif (!isset($this->installedModules[$module])) {
+            throw new \InvalidArgumentException(dt('Invalid @module module', [
+                '@module' => $module,
+            ]));
+        }
+    }
+
+    #[CLI\Command(name: self::WHY_CONFIG, aliases: ['wc'])]
+    #[CLI\Help(description: 'List all config entities depending on a given config entity')]
+    #[CLI\Argument(name: 'config', description: 'The config entity to check dependents for')]
+    #[CLI\Bootstrap(level: DrupalBootLevels::FULL)]
+    #[CLI\Usage(
+        name: 'drush why:config node.type.article',
+        description: 'Show all config entities modules depending on node.type.article'
+    )]
+    #[CLI\Usage(
+        name: 'drush why:config node.type.article --format=yaml',
+        description: 'Return config entity dependents as YAML'
+    )]
+    #[CLI\Topics(topics: [DocsCommands::DRUPAL_DEPENDENCIES])]
+    public function dependentsOfConfig(string $config, array $options = [
+        'format' => '',
+    ]): string|UnstructuredData|null
+    {
+        $this->scanConfigs(false);
+        $this->buildDependents($this->dependencies['config-config']);
+
+        if (!isset($this->dependents[$config])) {
+            $this->logger()->notice(dt('No other config entity depends on @config', [
+                '@config' => $config,
+            ]));
+            return null;
+        }
+
+        $this->buildTree($config);
+
+        if (empty($options['format'])) {
+            return $this->drawTree($config);
+        }
+
+        return new UnstructuredData($this->tree);
+    }
+
+    #[CLI\Hook(type: HookManager::ARGUMENT_VALIDATOR, target: 'why:config')]
+    public function validateDependentsOfConfig(CommandData $commandData): void
+    {
+        $configName = $commandData->input()->getArgument('config');
+        $configManager = \Drupal::getContainer()->get('config.manager');
+        if (!$configManager->loadConfigEntityByName($configName)) {
+            throw new \InvalidArgumentException(dt('Invalid @config config entity', [
+                '@config' => $configName,
+            ]));
+        }
+    }
+
+    /**
+     * @param string $dependency
+     * @param array $path
+     */
```

Comment:
Lets do a 1 line summary of the method, and omit phpdoc on params if they add nothing above whats in the function signature. Same for methods below.